### PR TITLE
Add jsonstore.io

### PR DESCRIPTION
**Pull request by bluzi**
**Website:** https://www.jsonstore.io/

**Description:** 
jsonstore.io offers a free, secured and JSON based cloud datastore for small projects. You just enter https://www.jsonstore.io/, copy the URL and start sending HTTP requests to communicate with your datastore.

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] This pull request has a descriptive title. For example, `Add [App name]`, not `Update readme.md` or `Added new entries`.
- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/awesome-no-login-web-apps/blob/master/CONTRIBUTING.md) and I am confident that my PR is suitable. 
- [x] The app I added is not a duplicate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -228,6 +228,7 @@ Powershell, Redis and lots others. Also has a good collection of IDEs for progra
 * [Codenvy](https://codenvy.com) `[Account]` - Cloud workspace for developers, supports most popular stacks including Android, Hadoop, Selenium, NodeJS and lots more.
 * [SQLFiddle](http://sqlfiddle.com/) - Tool for easy online testing and sharing of database problems and their solutions.
 * [Brainfuck Visualizer](http://fatiherikli.github.io/brainfuck-visualizer/) -  Javascript Brainfuck Interpreter + Visualizer. Helps you understanding how code is working.
+* [
```
Comment (aviaryan) [inline README.md]: Can you please remove this change?

File: README.md [hunk 2]
```diff
@@ -248,6 +249,7 @@ Powershell, Redis and lots others. Also has a good collection of IDEs for progra
 * [CSS Autoprefixer](https://autoprefixer.github.io/) - A postprocessor for handling vendor prefixes in CSS. Just paste the regular CSS and get the vendor prefix handled CSS instantly.
 * [CSS Matic](https://www.cssmatic.com/) - CSS tools to quickly generate gradients, border radius, noise textures and box shadow and get corresponding CSS or SASS.
 * [Carbon](https://carbon.now.sh) - Create and share beautiful images of your source code.
+* [jsonstore.io](https://www.jsonstore.io/) - Free cloud datastore.
```
Comment (aviaryan) [inline README.md]: Please add more detail here. For example, this description is more meaningful:

```
jsonstore.io offers a free, secured JSON based API endpoints for small projects. It supports all types of operations like POST, GET, PUT, DELETE etc
```

File: README.md (hunk 1 above)
Comment (bluzi) [reply README.md]: Sure, no idea how it got there O.o

**Comment by bluzi** [comment]
Made the change, wdyt? 
I removed the part about the POST, DELETE, etc.. as it seems obvious (every datastore supports querying, posting and deleting entries) 

If you still feel like it needs more context I can try again :P

**Comment by aviaryan** [comment]
Thanks, @bluzi.
And congrats on `jsonstore`'s success.

**Pull request merged**